deploy-board/deploy_board/templates/groups/asg_config.html

```diff
@@ -26,7 +25,14 @@
 
 {% block main %}
 
-
+{% if storage %}
+    {% for message in storage %}
+        <div class="alert alert-warning" role="alert">
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            <strong>Warning!</strong> {{ message | safe}}
+        </div>
+    {% endfor %}
+{% endif %}
```

Comment:
Isn't this block similar to the `message_banner.tmpl`? Would it still work you include it in this file and rename "storage" to "messages" on line 1132 in group_view.py?